kex.c

```diff
@@ -154,6 +154,18 @@ kex_alg_list(char sep)
 	return ret;
 }
 
+char *
+kex_alg_list(char sep)
+{
+	return kex_alg_list_internal(sep, kexalgs);
+}
+
+char *
+kex_gss_alg_list(char sep)
+{
+	return kex_alg_list_internal(sep, kexalg_prefixes);
```

Comment:
Maybe rename this to `gss_kexalgs` or thereabouts as you suggested earlier, while you're here?

Reply:
I was not sure whether the naming `gss_kexalgs` would be better since I did not hear from you back in #2. But probably yes. I will go with this name.

Reply:
Yep, sorry, I missed that at the time.  It seems better particularly in combination with this change.